js/lib/fixed_res_buffer.js

```diff
@@ -45,7 +46,6 @@ var FRBView = widgets.DOMWidgetView.extend({
         this.ctx.imageSmoothingEnabled = false;
         this.model.on('change:width', this.width_changed, this);
         this.model.on('change:height', this.height_changed, this);
-        yt_tools.booted.then(function(yt_tools) {
             this.colormaps = this.model.get('colormaps');
```

Comment:
Is the indentation required for these lines now that the promise happens earlier?

Reply:
Nope!  Thanks.